src/gasman.h

```diff
@@ -388,8 +388,14 @@ void CHANGED_BAG(Bag b);
 
 extern Bag * YoungBags;
 extern Bag   ChangedBags;
+
+#ifdef GAP_KERNEL_DEBUG
+BOOL         InWorkspaceRestore(void);
```

Comment:
```suggestion
BOOL InWorkspaceRestore(void);
```

Reply:
The place where I've put it is where clang-format wants to put it (I guess to line us with the variables above?), so I'd prefer to leave it there, else it might just move in future if we ever clang-format everything.

Reply:
Makes sense. I thought it would be the other way around